drivers/gpu/drm/bridge/sn65dsi84/sn65dsi84.c

```diff
@@ -351,6 +352,24 @@ static int sn65dsi84_get_modes(struct sn65dsi84 *sn65dsi84,
 	drm_mode_connector_update_edid_property(connector, (sn65dsi84->edid));
 	count = drm_add_edid_modes(connector, (sn65dsi84->edid));
 
+    if (sn65dsi84->i2c_edid)
+    {
```

Comment:
I suggest moving the  '{' on line 356 to line 355 to match the code style of the source.